docs/0.2/creating_products.md

```diff
@@ -139,48 +142,27 @@ Set the following Grid environment variables to specify information for the
 Each product requires an organization ID (the owner) and at least one agent
 with permission to create, update, and delete the product. There must also
 be an existing product schema for the product's properties.
-See [Prerequisites](#prerequisites) for more information.
+See [Prerequisites](#prerequisites) for more information. An example file
+defining GDSN product information can be downloaded with this link:
+<a href="/docs/0.2/references/product/example_product.xml"
+download="product.xml">GDSN product data</a>.
 
 {:start="5"}
+5. Use `docker cp` to copy your product data XML file into the `gridd-alpha`
+container.
 
-5. Create a product definition file, in YAML format, that specifies the
-   following information:
-
-   * Product namespace (such as GS1) that corresponds with the product schema
-     name (such as `gs1_product`).
-   * Product ID (such as a GTIN)
-   * Owner (organization ID)
-   * Set of product properties (each with a name, data type, and value
-     that conforms to the data type)
-     <br><br>
-
-   ```yaml
-   - product_namespace: "GS1"
-     product_id: "013600000929"
-     owner: "myorg"
-     properties:
-      product_name: "Truvia 80 ct."
-      image_url: "https://target.scene7.com/is/image/Target/GUEST_b7a6e983-b391-40a5-ad89-2f906bce5743?fmt=png&wid=1400&qlt=80"
-      brand_name: "Truvia"
-      product_description: "Truvia Sugar 80CT"
-      gpc: 30016951
-      net_content: "80CT"
-      target_market: 840
    ```
-
-   Tip: Use a YAML linter to validate the new file is formatted correctly.
-
-1. Use `docker cp` to copy the file into the `gridd-alpha` container.
-
-   ```
-   $ docker cp product.yaml gridd-alpha:/
+   $ docker cp product.xml gridd-alpha:/ \
+   && docker cp additional_attributes.yaml gridd-alpha:/
```

Comment:
Move the copying of the additional attributes to the "Defining additional attributes" section